test/abi/macOS/arm64/stdlib.swift

```diff
@@ -933,6 +933,8 @@ Added: _$ss10ArraySliceV11mutableSpans07MutableD0VyxGvr
 Added: _$ss15ContiguousArrayV11mutableSpans07MutableD0VyxGvr
 Added: _$ss11InlineArrayVsRi__rlE11mutableSpans07MutableD0Vyq_Gvr
 Added: _$ss15CollectionOfOneV11mutableSpans07MutableE0VyxGvr
+Added: _$sSrsRi_zrlE11mutableSpans07MutableB0VyxGvr
+Added: _$sSw12mutableBytess14MutableRawSpanVvr
```

Comment:
Hmm. Could the property itself have been `@_alwaysEmitIntoClient` to avoid these symbols, or no?

Reply:
Unfortunately computed properties for escapable types produce symbols to support keypaths. This is fixed for non-escapable types: their computed properties do not produce symbols.

Reply:
@glessard can you follow up with a PR to do that?

Reply:
Of course! I've started searching for how many other span/mutableSpan/etc symbols I missed this way…

Reply:
Note: followup [here](https://github.com/swiftlang/swift/pull/82517).